package.json

```diff
@@ -24,38 +24,44 @@
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
   ],
+  "types": "types/index.d.ts",
   "dependencies": {
     "ccount": "^1.0.3",
+    "dtslint": "^3.5.2",
```

Comment:
Could `dtslint` be moved to `devDependencies`?

Reply:
Done.

I don’t know why I keep making this exact mistake over and over :thinking: